pep/app/services/current-user.ts

```diff
@@ -255,4 +268,87 @@ export default class CurrentUserService extends Service {
         const prefs = this.preferences;
         return prefs?.[key] ?? [];
     }
+
+    /**
+     * Set new font size on html element
+     *
+     * @param {FontSizes} newSize
+     * @memberof CurrentUserService
+     */
+    setFontSize(newSize: FontSizes) {
+        const document = this.document;
+        let target = document.documentElement;
+        const size = AvailableFontSizes.find((item) => item.id === newSize) ?? FONT_SIZE_DEFAULT;
+        AvailableFontSizes.map((item) => item.class).forEach((item) => {
+            removeClass(target, item);
+        });
+
+        addClass(target, size.class);
+    }
+
+    /**
+     * Update the font size by saving to the user preferences
+     *
+     * @param {FontSizes} newSize
+     * @memberof CurrentUserService
+     */
+    updateFontSize(newSize: FontSizes) {
+        this.updatePrefs({ [PreferenceKey.FONT_SIZE]: newSize });
+    }
+
+    /**
+     * Update the text justification by saving to the user preferences
+     *
+     * @param {TextJustificationId} textJustification
+     * @memberof CurrentUserService
+     */
+    updateTextJustification(textJustification: TextJustificationId) {
+        this.updatePrefs({ [PreferenceKey.TEXT_JUSTIFICATION]: textJustification });
+    }
+
+    /**
+     * Available text justification options
+     *
+     * @readonly
+     * @memberof CurrentUserService
+     */
+    get availableTextJustifications() {
```

Comment:
can we move the getter declarations above the class methods, to keep in line with our formatting conventions